core/tauri/src/app.rs

```diff
@@ -1527,6 +1527,24 @@ impl<R: Runtime> Builder<R> {
   }
 }
 
+#[cfg(target_os = "macos")]
+fn init_app_menu<R: Runtime>(menu: &Menu<R>) -> crate::Result<()> {
+  menu.inner().init_for_nsapp();
+
+  if let Some(window_menu) = menu.get(crate::menu::WINDOW_SUBMENU_ID) {
+    if let Some(m) = window_menu.as_submenu() {
+      m.set_as_windows_menu_for_nsapp()?;
+    }
+  }
+  if let Some(help_menu) = menu.get(crate::menu::HELP_SUBMENU_ID) {
```

Comment:
why do we need to add this? can't we just let the user call the API manually if they need to?

Reply:
yeah but it's a nice convenient way of handling it, plus they don't need to worry about when the API is called (seems like it must be done after the menu has been set to NSApp) and we need to do something like it anyway for the default menu.

Reply:
It is fine I guess, might need a bit more documentation in the future.